app/workers/check_organisation_links_worker.rb

```diff
@@ -1,13 +1,14 @@
 # Calls the Link Checker API to verify all links in public editions, either per organisation or not
 class CheckOrganisationLinksWorker
   include Sidekiq::Worker
+  ORGANISATION_EDITION_LIMIT = 25
 
   sidekiq_options queue: "link_checks"
 
   def perform(organisation_id)
     organisation = find_organisation(organisation_id)
 
-    public_editions(organisation).find_each do |edition|
+    public_editions(organisation).each do |edition|
```

Comment:
Can we not use `find_each` here?

Reply:
Under the hood `find_each` uses `find_in_batches` which ignores the `limit` 

> You can’t set the limit either, that’s used to control the the batch sizes.
https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_in_batches